csv/csv_parse_stream.ts

```diff
@@ -87,7 +87,7 @@ export class CsvParseStream<
 
   #headers: readonly string[] = [];
 
-  constructor(options: T = defaultReadOptions as T) {
+  constructor(options?: T) {
     this.#options = {
       ...defaultReadOptions,
       ...options,
```

Comment:
defaultReadOptions is set on line 92, so there is no need to use it as a default value for the constructor here.